e2e/build.gradle

```diff
@@ -0,0 +1,40 @@
+subprojects {
+    buildscript {
+        repositories {
+            maven {
+                url "https://plugins.gradle.org/m2/"
+            }
+        }
+        dependencies {
+            classpath 'com.bmuschko:gradle-docker-plugin:7.0.0'
+        }
+    }
+
+    sourceSets {
+        integrationTest {
+            java {
+                compileClasspath += main.output + test.output
+                runtimeClasspath += main.output + test.output
+                srcDir file('src/integrationTest/java')
+            }
+            resources.srcDir file('src/integrationTest/resources')
+        }
+    }
+
+    configurations {
+        integrationTestImplementation.extendsFrom testImplementation
+        integrationTestRuntime.extendsFrom testRuntime
+    }
+
+    def DATA_PREPPER_CORE_JAR = project(':data-prepper-core').jar
```

Comment:
I try to avoid `def`s in Gradle builds. Is there a way to clean this up?

Reply:
I think it is not bad practice since it appeared in docs: https://docs.gradle.org/current/userguide/more_about_tasks.html

Reply:
I don't think it is bad practice, but I do try to avoid them when possible. But, if there isn't a better way, that is fine too.

Also, this isn't a constant, so it should probably also be camelCase: `dataPrepperCoreJar`.